Dockerfile

```diff
@@ -24,4 +24,8 @@ RUN mkdir -p /go/src/github.com/alecthomas && \
     go get -u golang.org/x/tools/cmd/goimports && \
     mkdir -p /go/src/github.com/mesosphere/csilvm
 
+# We explicitly disable use of lvmetad as the cache appears to yield inconsistent results,
+# at least when running in docker.
```

Comment:
"at least when running in docker" .. but not IRL? curious as to how this setting will impact people in production

Reply:
Yep, on my laptop I had issues with lvmetad when running inside docker, but the same tests pass when I run them normally (with lvmetad enabled).